db/app/controllers/topic-controllers.js

```diff
@@ -0,0 +1,12 @@
+module.exports = { getTopics };
+const {getAllTopics} = require('../models/topic-models')
+
+
+function getTopics(req, res, next) {
+  return getAllTopics()
```

Comment:
These function names are confusing. getTopics in the controller  is nice and clear but your models function should be something like selectAllTopics instead.